### DISCUSSION TITLE
Commits from the merge queue should be also shown in the main branch

### DISCUSSION THREAD

**Discussion by justinchuby** [codecov/feedback#98] [Ideas]
We recently enabled codecov for the onnx/onnx repository. One of the things we discover is there is no "main" branch from the drop down:

![image](https://github.com/codecov/feedback/assets/11205048/7e72a29e-1873-484f-a91d-7ba5a8bf9e85)

Turns out the commits from the merge queue branch `gh-readonly-queue/*` are the same commits in `main` (e.g. https://app.codecov.io/gh/onnx/onnx/commit/b61d2a8ab7a7e60eb386a387c73e449dd3461b3e), but codecov does not show these commits as part of the main branch. This causes the portal to show no "main" branch at all for coverage data, which can be inconvenient and confusing.

**Comment by rohan-at-sentry** [comment]
@justinchuby I created https://github.com/codecov/feedback/issues/216 so we can investigate.
